__tests__/endpoint/seed.test.js

```diff
@@ -20,14 +20,33 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((response) => {
-        expect(response.body.topics).toHaveLength(3)
+        expect(response.body.topics).toHaveLength(3);
         response.body.topics.forEach((topic) => {
           expect(typeof topic.slug).toBe("string");
           expect(typeof topic.description).toBe("string");
         });
       });
   });
+});
+
+describe("/api", () => {
+  test("should GET: 200 responds with an object which describes all the endpoints on this API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const parsedResponse = JSON.parse(response.text);
+        const vals = Object.values(parsedResponse);
+        vals.forEach((obj) => {
```

Comment:
A good test, but keep in mind that you can require in the JSON file directly, then use that required in variable to make your test assertions! Try require in the file and log what it gives you for for experimentation...